Running test on one file added in README.md (#876)
publiclab/mapknitter: README.md

`rake test:all`

> #### To run one test file:

Reviewer: ```suggestion
#### Running tests of a specific file:
```

`rails test test/unit/some_file.rb`

> #### And to run just a single test within a file:

Reviewer: ```suggestion
#### Running a single test from the test suite:
```
